### DISCUSSION TITLE
[General] <Bridging Error when trying to bridge a new ERC-20 Asset from Ethereum to ZK Sync Era>

### DISCUSSION THREAD

**Discussion by Griptonite** [zkSync-Community-Hub/zksync-developers#725] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Was trying to bridge some deUSD in from Ethereum to zkSync Era earlier and got this error message:

An internal error was received.

Details: Transaction does not have a transaction hash, there was a problem
Version: viem@2.7.16

Tried bridging a second time an hour later and it went through, so not sure what the issue was the first time around.

**Comment by alhenaai** [comment]
It seems like the initial bridging attempt you experienced may have been due to a temporary issue or glitch in the bridging process. Since your second attempt was successful, this indicates that the issue was resolved. However, if you encounter similar problems in the future or need more detailed insights into what might have caused the error, I recommend reaching out to the support team of the bridge you used, especially if it was a third-party bridge.

For any further assistance or if you have more questions regarding transactions on ZKsync, feel free to reach out.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by dutterbutter** [comment]
@Griptonite could you provide reproduction steps here? If it was intermittent as described it could be a result of the RPC used.

**Comment by dutterbutter** [comment]
Closing as I dont think there is any resolution here.
